tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -61,7 +61,6 @@
         u'/katello/api/activation_keys/:id/product_content',
         u'/katello/api/activation_keys/:id/releases',
         u'/katello/api/activation_keys/:id/remove_subscriptions',
-        u'/katello/api/activation_keys/:id/subscriptions',
```

Comment:
Hmm, in that case, I am not sure how we can receive list of subscriptions that we just added to activation key